recipes-kernel/linux/linux-starfive_6.4.bb

```diff
@@ -0,0 +1,25 @@
+require linux-mainline-common.inc
+FILESEXTRAPATHS =. "${FILE_DIRNAME}/linux-starfive:"
+SUMMARY = "VisionFive 6.4.y Linux Kernel recipe"
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
+KERNEL_VERSION_SANITY_SKIP = "1"
+
+SRCREV = "92f369569bee04137e14c376cafbe23bdfb6474a"
+
+SRC_URI = " \
+	git://github.com/starfive-tech/linux.git;protocol=https;branch=visionfive-6.4.y \
+	file://0001-gcc-plugins-Rename-last_stmt-for-GCC-14.patch \
+	file://modules.cfg \
+"
+
+SRC_URI:append:beaglev-starlight-jh7100 = " \
+	file://extra.cfg \
+"
+
+LINUX_VERSION ?= "6.4.4"
+LINUX_VERSION_EXTENSION:append:beaglev-starlight-jh7100 = "-starlight"
+
+KBUILD_DEFCONFIG:beaglev-starlight-jh7100 = "starfive_jh7100_fedora_defconfig"
+
+COMPATIBLE_MACHINE = "(beaglev-starlight-jh7100)"
```

Comment:
I think this can be adjusted in the `linux-starfive-dev` recipe too, I think if we can do so, it will be nice, we dont have to maintain another recipe for beagleV, can you look into this aspect ?

Reply:
Actually, I adjusted it in **linux-starfive-dev** at first but then I thought it was a release, not a dev branch, so I created **6.4** a separate recipe.

Considering your comment, seems like it's okay to use release branch in **linux-starfive-dev** so I will update this.